amoro-ams/src/main/java/org/apache/amoro/server/resource/DefaultOptimizerManager.java

```diff
@@ -40,8 +49,15 @@
 public class DefaultOptimizerManager extends PersistentBase implements OptimizerManager {
 
   protected final Configurations serverConfiguration;
-
-  public DefaultOptimizerManager(Configurations serverConfiguration) {
+  private final CatalogManager catalogManager;
+  private final MaintainedTableManager tableManager;
```

Comment:
it seems the `tableManager` does not need here

Reply:
Yes, it can be dropped.